lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -759,13 +760,12 @@ sub _failed_check {
                    size => 15,
            }};
         push @$rows, $row;
-        $hiddens->{"id_$count"} = $row->{$check->id_column};
         ++$count;
+        $hiddens->{"id_$count"} = $row->{$check->id_column};
```

Comment:
You need to move the count to before this line because you made the count 'zero-based' above. However, I don't think you need either change. In case the count needs to be zero-based, please add a comment as to why that is in the code.